CHANGELOG.md

```diff
@@ -0,0 +1,986 @@
+# Changelog
```

Comment:
I loaded this up in my editor (VSCode) which has the [markdownlint plugin](https://github.com/DavidAnson/vscode-markdownlint) installed with all default settings. The default linter settings include:
- MD022:  blanks-around-headings/blanks-around-headers - Headings should be surrounded by blank lines
- MD024: no-duplicate-heading/no-duplicate-header - Multiple headings with the same content 
- MD032: blanks-around-lists - Lists should be surrounded by blank lines

I was able to clear up these warnings, since they're desired formatting according to your PR description, by adding the following line to the top of the file before the `# Changelog` line:
```
<!-- markdownlint-disable MD022 MD024 MD032 -->
```
Up to you if you want to add this line since we don't explicitly enforce or run a linter on this file in CI or elsewhere, but I figured I'd mention it since it helps to clean up the formatting.

Reply:
Building on the previous comment, I ran the markdown auto formatter after applying the line above and got the following diff. Also up to you if you want to apply it but it fixes a few missed formatting issues for consistency throughout the document:

[changelog-md-lint-diff.txt](https://github.com/locationtech/geotrellis/files/3755916/changelog-md-lint-diff.txt)

It mostly changes list indentation and code block formatting.